### ISSUE TITLE
Strange windows' sizes at wide frames

### ISSUE THREAD

**Issue by x4lldux** [roman/golden-ratio.el#40]
There isn't a better way to describe it than by a screenshots:
![zrzut ekranu z 2014-12-21 17 20 48](https://cloud.githubusercontent.com/assets/22363/5518776/c513bc70-8935-11e4-9afb-9e333b4dbcd1.png)
![zrzut ekranu z 2014-12-21 17 18 56](https://cloud.githubusercontent.com/assets/22363/5518777/cb39359e-8935-11e4-90ee-1d77c9adf364.png)
![zrzut ekranu z 2014-12-21 17 20 21](https://cloud.githubusercontent.com/assets/22363/5518778/d5059f86-8935-11e4-9a84-4ee552da45ec.png)

Versions:
GNU Emacs 24.4.1 (x86_64-suse-linux-gnu, GTK+ Version 3.14.5) of 2014-11-26 on cloud127
golden-ratio.el: 6173b7c4165b056edbbd948d890dd3498f78138b

Please tell me if you need some more info!

**Comment by stugots** [comment]
I see this too.  GNO Emacs 24.4.1 on OS X.

**Comment by thierryvolpiatto** [comment]
> There isn't a better way to describe it than by a screenshots.

Not really, this is probaly the reason why you had no answers yet, please be more clear, explaining what is wrong in each screenshot.

**Comment by stugots** [comment]
> Not really, this is probaly the reason why you had no answers yet, please be more clear, explaining what is wrong in each screenshot.

On wide displays, the screen widths are dramatically different between the active windows and the other windows. The other windows are far too narrow.

Use it on a 27" or 32" display yourself.

**Comment by vmalloc** [comment]
+1 getting this as well

**Comment by X4lldux** [comment]
@vmalloc @stugots setting `window-combination-resize` to non-nil fixed it for me.

**Comment by angrybacon** [comment]
I am reproducing this issue on a wide display as well, reglardless of `golden-ratio-adjust-factor`'s value. You can see the issue on the screenshots below:

- First picture: bottom-right window is active. Everything is as expected
- Second picture: top-right window is active. The inactive windows are no longer equally split.

However, @X4lldux's workaround (solution?) fixes this.

Below 
![screen shot 2016-04-20 at 11 59 09 2](https://cloud.githubusercontent.com/assets/5608624/14785775/ade7afda-0afc-11e6-83bd-a1b1b3faea27.png)
![screen shot 2016-04-20 at 11 58 35 2](https://cloud.githubusercontent.com/assets/5608624/14785780/b10d9210-0afc-11e6-829e-b554dda6490e.png)

**Comment by ZelphirKaltstahl** [comment]
Behavior still persists in `GNU Emacs 29.1`.

**Comment by ZelphirKaltstahl** [comment]
As I understand it, the expectation is, that the "active" window gets the bigger portion of the golden ratio split, and the "inactive" windows evenly split the remaining space.

**Comment by ZelphirKaltstahl** [comment]
~Can confirm: Setting `(setq window-combination-resize t)` does fix it.~

Edit: Nevermind, apparently it can still be confused by simply switching around between windows a bit and then one is back to too narrow columns.
